votes/0546-tisonkun-as-tidb-reviewer.md

```diff
@@ -0,0 +1,22 @@
+# A Vote for tisonkun as TiDB Reviewer
+
+## Proposal
+
+[@tisonkun](https://github.com/tisonkun) has been working on `pingcap/tidb` for several months. IMO he made a lot of great contribution to TiDB, including the tracking of unstable tests(#25899), restructing test cases(#26022). What's more, he shows a great vision to the TiDB community. The details of his contributions are listed as follow:
+
+* [Authored pull requests](https://github.com/pingcap/tidb/commits?author=tisonkun).
+* [Reviewed pull requests](https://github.com/pingcap/tidb/pulls?q=is%3Apr+reviewed-by%3Atisonkun)
+
+I (@bb7133) hereby nominate @tisonkun as TiDB Reviewer and call for a vote.
+
+## Deadline
+
+The vote will be open for at least 2 days unless there is an objection or not enough votes.
```

Comment:
```suggestion
The vote will be open for at least 3 days unless there is an objection or not enough votes.
```

according to https://github.com/pingcap/community/blob/master/teams/README.md#actions